app/views/events/show.html.erb

```diff
@@ -100,13 +99,29 @@
               leave_event_path(@event),
               method: :put,
               class: 'btn btn-default',
+              disabled: (not @event.ownes_participating_teams?(current_user)),
               id: :leave_event_button %>
   <%= t('events.participating') %>
 <% end %>
-<% if can? :join, @event %>
-  <%= link_to t('.join', default: t('helpers.links.join')),
-              join_event_path(@event),
-              method: :put,
-              class: 'btn btn-default',
-              id: :join_event_button %>
+<% if can? :join, @event%>
+  <% if @event.single? %>
+    <%= link_to t('.join', default: t('helpers.links.join')),
```

Comment:
is this differenciation really necessary? referring to a comment I made above: Where is my single-player-team